.travis.yml

```diff
@@ -3,7 +3,10 @@ cache: bundler
 language: ruby
 dist: trusty
 rvm:
-  - jruby-9.0.1.0
+  - jruby-9.0.5.0
+  - jruby-9.1.9.0
+  - jruby-head
+  - jruby-1.7.26
```

Comment:
I think we don't support jruby-1.7.x https://github.com/bbatsov/rubocop#compatibility

Reply:
Thanks, I removed it from the list.